docs/section6/Rule6-9.md

```diff
@@ -0,0 +1,72 @@
+
+# Lighting - Rule 6-9
+
+**Rule ID:** 6-9  
+**Rule Description:** Baseline building is modeled with automatic shutoff controls in buildings >5000 sq.ft.  
+**Appendix G Section:** Section G3.1-6 Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:**  None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+  1. Building total area is more than 5,000sq.ft.  
+
+**Manual Check:** Yes  
+
+**Function Call:**  
+
+  - compare_schedules()
+  - normalize_space_lighting_schedule()
+
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic: 
+
+- For each building_segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`  
+
+  - For each thermal_block in building segment: `for thermal_block_b in building_segment.thermal_blocks:`  
+
+    - For each zone in thermal block: `for zone_b in thermal_block_b.zones:`  
+
+      - For each space in zone: `for space_b in zone.spaces:`  
+
+        - Add space floor area to building total floor area: `building_total_area_b += space_b.floor_area`  
+
+- **Applicability Check 1:**`if building_total_area_b > 5000:`  
+
+- Get building open schedule in the baseline model: `building_open_schedule_b = B_RMR.building.building_open_schedule`  
+
```

Comment:
I think we discussed adding BuildingSegment.open_schedule, because I believe lighting shut-off controls must be included for each occupancy type and not building as a whole.

Reply:
yes, this is a placeholder till that is discussed by the schema WG and added to the schema.
@xingcx maybe you could include a note towards the end on future revisions. This RDS should reference buildingsegment.openschedule once that is discussed, approved and added to the schema

Reply:
Note added.